Does not build using GCC 8.2.1

Using stock GCC 8.2.1 on Fedora 29, the library doesn't build properly as it runs into warning and all warnings are treated as errors:

```
[skade@Nostalgia-For-Infinity vme]$ gcc --version
gcc (GCC) 8.2.1 20181215 (Red Hat 8.2.1-6)
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

```
[skade@Nostalgia-For-Infinity vme]$ make
gcc -g -Wall -Werror -std=gnu99 -O2 -fPIC   -c -o vantiq_client.o vantiq_client.c
vantiq_client.c: In function ‘vc_init’:
vantiq_client.c:272:9: error: ‘strncpy’ specified bound depends on the length of the source argument [-Werror=stringop-overflow=]
         strncpy(vc->server_url, url, len-1);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
vantiq_client.c:268:18: note: length computed here
     size_t len = strlen(url);
                  ^~~~~~~~~~~
cc1: all warnings being treated as errors
make: *** [<builtin>: vantiq_client.o] Error 1
```

--- a/src/vme/vantiq_client.c
+++ b/src/vme/vantiq_client.c
@@ -89,7 +89,7 @@ static size_t header_callback(char *buffer, size_t size, size_t nitems, void *us
     size_t headerNameSz = sizeof(COUNT_HEADER)-1;
     /* received header is nitems * size long in 'buffer' NOT ZERO TERMINATED */
     /* 'userdata' is set with CURLOPT_HEADERDATA */
-    if (len > headerNameSz && memcmp(buffer, COUNT_HEADER, headerNameSz) == 0) {
+    if (len > headerNameSz && strncasecmp(buffer, COUNT_HEADER, headerNameSz) == 0) {
         char countBuf[32];
         int pos = (int)headerNameSz;
         memcpy(countBuf, buffer + pos, len - pos);
@@ -269,7 +269,7 @@ vantiq_client_t *vc_init(const char *url, const char *authToken, uint8_t apiVers
     if (url[len-1] == '/') {
         // trim trailing slash
         vc->server_url = (char *)malloc(len);
-        strncpy(vc->server_url, url, len-1);
+        memcpy(vc->server_url, url, len-1);
         vc->server_url[len-1] = '\0';
     } else {
         vc->server_url = strdup(url);

--- a/src/vmeTest/config.properties
+++ b/src/vmeTest/config.properties
@@ -2,5 +2,5 @@
 #DPIPORT = 3490
 DPISOCKETPATH = /tmp/vipo_socket
 VANTIQ_BASEURL = https://dev.vantiq.com
-VANTIQTOKEN = KtaUNyWsplL-a6Lujaghp7Ky7_v4JfI1Et7LQLunwqI=
-#LOG_LEVEL = DEBUG
+VANTIQTOKEN = <replace with Vantiq access token>
+# LOG_LEVEL = DEBUG

